.github/workflows/build-docker-images.yml

```diff
@@ -307,3 +307,41 @@ jobs:
   #           REF=main
   #         push: true
   #         tags: huggingface/transformers-pytorch-deepspeed-amd-gpu-push-ci
+
+  latest-quantization-torch-docker:
+    name: "Latest Pytorch + Quantization [dev]"
+     # Push CI doesn't need this image
+    if: inputs.image_postfix != '-push-ci'
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Cleanup disk
+        run: |
+          sudo ls -l /usr/local/lib/
+          sudo ls -l /usr/share/
+          sudo du -sh /usr/local/lib/
+          sudo du -sh /usr/share/
+          sudo rm -rf /usr/local/lib/android
+          sudo rm -rf /usr/share/dotnet
+          sudo du -sh /usr/local/lib/
+          sudo du -sh /usr/share/
```

Comment:
we can remove this now I guess (once we use `[intel-cpu, 8-cpu, ci]`)

Reply:
removed !